CONTRIBUTING.md

```diff
@@ -75,6 +75,11 @@ Use the build scripts provided in the `ci/` directory to build tests for each co
 ```bash
 ./ci/build_cub.sh -cxx g++ -std 14 -arch "70;75;80-virtual"
 ```
+Once you have run the script, and your configuration is properly set up, you can simply use `CMake` to build your targets which will be much faster than the CI scripts when checking to see if your code compiles. For example, to build `CUB` locally with C++ 17, you can run the following command.
```

Comment:
question: What makes using the preset faster than the build script?

Reply:
I haven't done a deep enough dive on what the scripts are doing internally so I can't comment on the difference, but I've noticed empirically that it is much slower than just directly using the preset. 

The feedback loop was really slow for me personally. If I made a small syntactical error in writing my code, it would take 2-3 minutes from running the build scripts before I got to see the error. That compounds really hard, where as running the preset build directly the feedback is nearly instantaneous.

Reply:
> take 2-3 minutes from running the build scripts
> That compounds really hard, where as running the preset build directly the feedback is nearly instantaneous.

That's good to know, thanks for raising this. I completely agree with not wanting to use the scripts if this is the experience you're having!

This is definitely not the behavior I would expect, so we'll need to investigate why this is happening for you.